tasks/main.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ capra_lvm_filesystem_packages }}"
+
+- name: "Configure default mountpoint directory"
+  file:
+    path: "{{ capra_lvm_filesystem_mount_parent }}"
+    state: directory
+    owner: "{{ capra_lvm_filesystem_owner }}"
+    group: "{{ capra_lvm_filesystem_group }}"
+    mode: 0755
+    seuser: unconfined_u
+    serole: object_r
+    setype: default_t
+    selevel: s0
+
+- name: "Unmount volume"
+  mount:
+    path: "{{ item.mount | default(capra_lvm_filesystem_mount_parent + '/' + item.name) }}"
+    state: absent
+  with_items: "{{ capra_lvm_filesystem_specs }}"
```

Comment:
It's a list, so `with_list` (applies to all other cases too).

Reply:
In order to remain compatible with Ansible 2.4 we have to use `with_items`. See https://docs.ansible.com/ansible/2.4/playbooks_loops.html.

Reply:
`with_list` is supported in 2.4 an widely used.